src/components/application_manager/src/event_engine/event_dispatcher_impl.cc

```diff
@@ -44,27 +44,27 @@ EventDispatcherImpl::EventDispatcherImpl() : observers_event_() {}
 EventDispatcherImpl::~EventDispatcherImpl() {}
 
 void EventDispatcherImpl::raise_event(const Event& event) {
+  ObserverVector observers;
```

Comment:
what is the reason to not use `observers_` member var? it seems this was the only use of observers_ besides `remove_observer_from_vector` and oddly `remove_mobile_observer_from_vector`

Reply:
@iCollin there was an issue with recursive `on_event` call when using `observers_` member.
You may check for details in [this PR](https://github.com/LuxoftSDL/sdl_core/pull/101)